src/lib/utils.ts

```diff
@@ -72,6 +72,22 @@ export function puzzleHash(address: string): string {
   return info.puzzleHash;
 }
 
+export function isValidAddress(address: string, prefix: string): boolean {
+  if (!address.startsWith(prefix)) {
```

Comment:
You can check the `addressInfo` output which has a `prefix` field too

Reply:
done